packages/integrations/solid/package.json

```diff
@@ -32,7 +32,8 @@
     "dev": "astro-scripts dev \"src/**/*.ts\""
   },
   "dependencies": {
-    "babel-preset-solid": "^1.4.2"
+    "babel-preset-solid": "^1.4.2",
+    "vitefu": "^0.1.0"
```

Comment:
Well this is interesting! I feel like we could replace our internal `astro` logic with this potentially, as well?

Reply:
Yes! I've already replaced that in #5203 😬

Reply:
Amazing!